web/documentserver-example/php/docker-compose.yml

```diff
@@ -3,7 +3,7 @@ version: "3.8"
 services:
   document-server:
     container_name: document-server
-    image: onlyoffice/documentserver:7.3.3.50
+    image: onlyoffice/documentserver:latest
```

Comment:
Again changes that aren't tied to the main idea of the request.

Using the latest version of any dependency is generally not recommended and should only be done in exceptional cases. In our situation, we manually choose the version of the Document Server to ensure that we can properly handle and test specific features of that particular release. Whenever a new release of the Document Server is published, we manually update the version in all examples. This helps us avoid unexpected errors and allows us to prepare for any new changes in the core product.

So, you must be very confident in yourself before pinning the dependency to the latest version.

Reply:
I have returned version 7.3.3.50. Updating the compose version to version 7.5 will be put in a separate request.